cmd/createtree/pem.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/google/trillian/crypto/keys"
+	"github.com/google/trillian/crypto/keyspb"
+)
+
+var (
+	pemKeyPath = flag.String("pem_key_path", "", "Path to the private key PEM file")
+	pemKeyPass = flag.String("pem_key_password", "", "Password of the private key PEM file")
+)
+
+func init() {
+	keyHandlers["PEMKeyFile"] = func() (proto.Message, error) {
```

Comment:
I would prefer keyHandlers to be declared and initialized in the same file. I would also prefer all flags to be defined together (at main.go), but since this is in the main package we're not breaking the rule too much.

Suggestion: for the sake of isolation / cleaner interface between createtree and key logic, we could do something like:

At createtree/keys/keys.go:

```go
package keys

type handler func() (proto.Message, error)

func SetHandler(name string, fn handler) { ... }

func Handler(name string) (handler, bool) { ... }

func NewPEMKeyHandler(path, pass) handler { ... }

```

At createtree/main.go:

```go
func main() {
    keys.RegisterHandler("PEMKeyFile", keys.NewPEMKeyHandler(*path, *pass))
    // ...
}
```

WDYT?